Automattic/jetpack
# Plans: add more variations on the Security plans' slugs.

#### Changes proposed in this Pull Request:

﻿This should fix issues with features that require a security plan, but where the new security plans were not recognized before.

This was reported in p8oabR-Nm-p2#comment-5918

#### Jetpack product discussion

* N/A

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* Start with a free site, connected to your WordPress.com account.
* Go to Posts > Add New, and type in `/Pay`; the Pay with PayPal block should not appear.
* Now go to Jetpack > Dashboard > Plans and purchase a Security plan.
* Upon coming back to your dashboard, head back to Posts > Add New; you should now have access to that block.
